testutil/ciao-down/download.go

```diff
@@ -88,25 +106,50 @@ func getUbuntu(ctx context.Context, dest io.WriteCloser, cb progressCB) (err err
 
 	buf := make([]byte, 1<<20)
 	_, err = io.CopyBuffer(dest, pr, buf)
-	_ = resp.Body.Close()
 
 	if err == nil && int(pr.downloaded/1000000)%10 != 0 {
-		downloadProgress(progress{downloadedMB: pr.totalMB, totalMB: pr.totalMB})
+		cb(progress{downloadedMB: pr.totalMB, totalMB: pr.totalMB})
 	}
 
 	return
 }
 
-func downloadUbuntu(ctx context.Context, instanceDir string, cb progressCB) (string, error) {
-	imgPath := path.Join(instanceDir, guestImageName)
+func downloadProgress(p progress) {
+	if p.totalMB >= 0 {
+		fmt.Printf("Downloaded %d MB of %d\n", p.downloadedMB, p.totalMB)
+	} else {
+		fmt.Printf("Downloaded %d MB\n", p.downloadedMB)
+	}
+}
+
+func downloadFile(ctx context.Context, URL, ciaoDir string, cb progressCB) (string, error) {
```

Comment:
@markdryan will there be issues here if we support multiple ciao-down instances in the future?